AjobK/Athena
# 337 mobx user to profile

closes #337 

<br />

 **What is it supposed to do**  
Safely deleted the user store from the MobX state tree. This is done without further adjustments, as it was not being used anywhere.

<br />

 **How can we test your branch**  
Doesn't change anything but you could try running the application to see if it still runs at all.

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist**

<br />

**4. If relevant, test these browsers**
